lib/application/workflow/cli/cli_handler.ex

```diff
@@ -0,0 +1,51 @@
+defmodule CLIHandler do
+  def handle_command(%CLICommand{command: directory, args: {_, options}}) do
```

Comment:
I think this isn't the right way to do it. The `command` should be something like an enum that says what is the command (e.g. `[:list_dir_size, :list_file_size]`), and the args should contain the relevant info to the command to work (e.g. the dir, the file, the optional arguments, etc).

Reply:
Done.